assets/js/base/utils/get-intersection-observer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Util that returns an IntersectionObserver if supported by the browser. If
+ * it's not supported, it returns a shim object with the methods to prevent JS
+ * errors. Notice it's a shim, not a polyfill. If the browser doesn't support
+ * IntersectionObserver, the methods returned by this function will do nothing.
+ *
+ * @param {IntersectionObserverCallback} callback Callback function for the
+ *                                                Intersection Observer.
```

Comment:
Maybe add a `@todo` comment here to remove this shim when we drop support for IE11?

Reply:
Done: b2e3d05ec6e9d371d3c8f2068cb86216133b669d. :slightly_smiling_face: